packages/localize/extension.package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "@theia/localize",
+  "version": "0.1.1",
+  "description": "Theia - Localize Extension to translate string int different languages.",
+  "dependencies": {
+    "cldr-data": "^31.0.2",
+    "globalize": "^1.3.0",
+    "@types/globalize": "0.0.30",
+    "@types/mocha": "^2.2.41",
+    "chai": "^4.1.0",
+    "chai-as-promised": "^7.1.1",
+    "globalize-webpack-plugin": "^1.1.1",
+    "shx": "^0.2.2"
+  },
+  "bin": {
+    "localize-theia": "bin/localize-theia"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "theiaExtensions": [
+    {
+      "frontend": "lib/localize-front-end-module"
```

Comment:
we usually call front-end `frontend` in one word

Reply:
Done. replace front-end by frontend